From 098c9dd1162ef3405213150fee88633b6e95b321 Mon Sep 17 00:00:00 2001
From: Gowtham Tammana <g-tammana@ti.com>
Date: Wed, 15 Jun 2016 16:04:52 -0500
Subject: [PATCH 39/89] export dmac_flush_range symbol

GC320 driver module throws below during link and module load
fails

WARNING: "v7_dma_flush_range" [galcore.ko] undefined!

Change-Id: I6618cfba31dabb8a78fc859b89edd49d365162f1
Signed-off-by: Gowtham Tammana <g-tammana@ti.com>
Signed-off-by: Jacob Stiffler <j-stiffler@ti.com>
---
 arch/arm/mm/proc-syms.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mm/proc-syms.c b/arch/arm/mm/proc-syms.c
index 054b491..778e32c 100644
--- a/arch/arm/mm/proc-syms.c
+++ b/arch/arm/mm/proc-syms.c
@@ -30,6 +30,7 @@ EXPORT_SYMBOL(__cpuc_flush_user_all);
 EXPORT_SYMBOL(__cpuc_flush_user_range);
 EXPORT_SYMBOL(__cpuc_coherent_kern_range);
 EXPORT_SYMBOL(__cpuc_flush_dcache_area);
+EXPORT_SYMBOL(dmac_flush_range);
 #else
 EXPORT_SYMBOL(cpu_cache);
 #endif
-- 
1.9.1

